tailcfg/derpmap.go

```diff
@@ -67,6 +67,17 @@ type DERPRegion struct {
 	// away to a new region without Avoid set.
 	Avoid bool `json:",omitempty"`
 
+	// Location is an (optional) geographic location for this region; it is
+	// used in conjunction with ambient location information like the
+	// current cloud provider's region to try optimal regions first and
+	// exclude DERP regions on the other side of the world from being
+	// checked.
+	//
+	// If this region has no location, it should be treated as always
+	// "close" to the current node–e.g. not excluded from netcheck probes
+	// by a location-based heuristic.
```

Comment:
I would say "If this region has no location, it should always be considered a candidate as maybe being the closest. That is, it should not be ruled out as a candidate if the node knows its own actual location."

BTW, Go standard library doc style is to avoid all Latin abbreviations. I'm not super picky about it, but I've come to kinda like it.

Reply:
Yeah, I like that wording; pushed 👍